api/integrations/fifabot/.eslintrc.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
```

Comment:
can't we use the same lint we have for the api project? Why do we have a different config file for lint in here?

Reply:
yes we can, i'll see how i can configure that.